atom/browser/api/event_subscriber.h

```diff
@@ -73,6 +73,13 @@ class EventSubscriber : internal::EventSubscriberBase {
           content::BrowserThread::UI, FROM_HERE,
           base::Bind(
               [](EventSubscriber<HandlerType>* subscriber) {
+                {
```

Comment:
Maybe it would be cleaner to do `ptr->handler_lock_.Acquire()` in the outer function and `ptr->handler_lock_.Release()` here?